app/lib/cfe_connection.rb

```diff
@@ -125,6 +125,8 @@ def api_result(assessment_id)
 
   def create_record(assessment_id, record_type, record_data)
     cfe_connection.post "/assessments/#{assessment_id}/#{record_type}", record_data
+  rescue Faraday::UnprocessableEntityError => e
+    raise e, "CFE returned the following message: #{e.response_body}", e.backtrace
```

Comment:
I wondering if it might be clearer just to turn off the raise_error Faraday middleware, and then check the result code for success here and then raise the custom exception. We could also add the URL of the request (or maybe just the record_type so show which call failed)